src/adapter/src/coord/timestamp_selection.rs

```diff
@@ -672,6 +835,8 @@ pub struct TimestampDetermination<T> {
     pub session_oracle_read_ts: Option<T>,
     /// The value of the real time recency timestamp, if used.
     pub real_time_recency_ts: Option<T>,
+    /// The constraints used by the constraint based solver.
```

Comment:
Adding to this comment about when the constraint based solver is used, or linking to the module where it's defined, e.g. 
```
See the [`constraints`] module for more details.
```
would be great!